test/c/eacsl/not-yet-supported/ghost_struct_field/field.t

```diff
@@ -1,2 +1,23 @@
   $ owi c --e-acsl ./field.c
-  All OK
+  [kernel] field.c:9: User Error: 
+    redefinition of 'List' in the same scope.
+    Previous declaration was at field.c:5
+  [kernel] field.c:17: User Error: 
+    Cannot find field order in type struct List
+    15            loop variant length;
+    16        */
+    
+    17        while (node != NULL) {
+    18            node = node->next;
+    19            //@ ghost length --;
+    20        }
+    
+    21    }
+    22
```

Comment:
suspicious errors ?

Reply:
The grammar for defining ghost struct fields is to define again the same structure in the ghost code.

```c
struct List {
    int element;
    struct List* next;
};
/*@ ghost struct List {
    int order;
}; */
```

It's not supported so the field seems undefined

Reply:
OK, maybe it should be good to open an issue on e-acsl to ask about changing this to a warning and to ignore `returns` ?